src/core/utils/tables.js

```diff
@@ -87,7 +87,19 @@ export const validConfigs = (config) => {
   }
 
   if (!config.mapping.subCounty) {
-    window.console.error('Invalid table config: mapping.subCounty is required!');
+    window.console.warn('Potentially invalid table config: mapping.subCounty may be required!');
+
+    return true;
+  }
+
+  if (config.dataID && !config.baseAPIUrl) {
+    window.console.error('Invalid table config: baseAPIUrl is required!');
+
+    return false;
+  }
+
+  if (!(config.data || config.url || config.dataID)) {
+    window.console.error('Invalid table config: one of data/url/dataID is required!');
```

Comment:
I see repetitions here, so no function?

Reply:
Hmm I don't see the repeatition, the error messages are different, and the config variables checked for are also different

Reply:
Ohh, I see

Reply:
There are two validation functions, one for tables and another for charts